ops/terraform/services/common/sqs.groovy

```diff
@@ -81,4 +80,27 @@ jq '.? | map({receipt: .ReceiptHandle, body: .Body | fromjson}) | unique'
     return jsonMessages
 }
 
+/* Wrapping `aws sqs send-message`, this sends a message to a given SQS queue
+ *
+ * @param args a {@link Map} must include sqsQueueUrl and sqsMessage
+ * <ul>
+ * <li>awsRegion targeted aws region. Defaults to 'us-east-1'</li>
+ * <li>sqsQueueUrl targeted sqs queue url</li>
+ * <li>sqsMessage string the message to send to the SQS queue</li>
+ * </ul>
+ */
+def sendMessage(Map args = [:]) {
+    awsRegion = args.awsRegion ?: 'us-east-1'
+    sqsQueueUrl = args.sqsQueueUrl
+    sqsMessage = args.sqsMessage
+
+    withEnv(["SQS_QUEUE_URL=${sqsQueueUrl}", "MESSAGE=${sqsMessage}"]) {
```

Comment:
I normally would not expect any issues when sending an SQS message, but I suppose it _is_ possible for there to be throttling or some other issue that causes this command to fail.

I've added retry logic to this function so that sending will be retried up to 15 times, waiting 2 seconds (by default) between each try. If the number of retries are exceeded, then the pipeline will error out.